runners_manager/start.py

```diff
@@ -1,14 +1,43 @@
-import os
+import argparse
 
-import runners_manager.settings.settings_local # noqa
+from runners_manager.settings.yaml_config import setup_settings
+# import runners_manager.settings.settings_local # noqa
 from runners_manager.main import main
 
 
-organization = os.getenv('GITHUB_ORGANIZATION')
+def setup_parser():
+    parser = argparse.ArgumentParser(add_help=True,
+                                     description='Start and manage github self-host runners')
+
+    parser.add_argument(
+        'settings_file',
+        default='setting.yaml',
+        help="Path to project settings file")
+    parser.add_argument(
+        'github_token',
+        help="Token for github API calls",
+    )
```

Comment:
are we expecting to now set those args through command line args? I would rather continue to see "secret configs" as environment variable.

We can still keep the option of having it in the args but I would suggest something like this:
```suggestion
    parser.add_argument(
        'github_token',
        default=os.getenv('GITHUB_TOKEN'),
        help="Token for github API calls",
    )
```
This argument is for every setting that is or works with a secret.